orbitx/physics.py

```diff
@@ -208,7 +208,7 @@ def set_state(self, physical_state):
         try:
             self._spacestation_index = [
                 entity.name for entity in physical_state.entities
-            ].index('SpaceStation')
+            ].index('AYSE')
```

Comment:
I think last time I saw this I asked if it could be replaced with PhysicsState['AYSE'] as an accessor, but you already know about that